torchprime/torch_xla_models/configs/task/basic.yaml

```diff
@@ -0,0 +1,11 @@
+name: basic  # Basic means simple forward/backward pass without any special task logic.
```

Comment:
nit: rename "basic" to "base" to indicate that this is a base functionality to be extended by SFT?